.travis.yml

```diff
@@ -0,0 +1,43 @@
+dist: bionic
+language: go
+
+services:
+  - docker
+
+go:
+  - 1.13.x
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - go: master
+
+go_import_path: github.com/Percona-Lab/promconfig
+
+# skip non-trunk PMM-XXXX branch builds, but still build pull requests
+branches:
+  except:
+    - /^PMM\-\d{4}/
+
+cache:
+  directories:
+    - /home/travis/.cache/go-build
+
+before_cache:
+  - go clean -testcache
+
+script:
+  - make ci
+
+after_success:
+  - curl https://codecov.io/bash > codecov
+  - chmod +x codecov
+  - ./codecov -f cover.out -F cover -X fix
+  - ./codecov -f crosscover.out -F crosscover -X fix
+
+#notifications:
+#  slack:
+#    on_success: change
+#    on_failure: always
+#    rooms:
+#      secure: Iy3M4BJaSmkyjwibcEW60i7R+PZJOPTgdvdTB6h7iN5uqqAjhRaA36ujyjyO1swjaP4/t8gJb0IKSd+fYhEedXAReTRPYsYkRDO0Z9MblwT/8XH43eMnRcEjvy1l3ovcOXcfYFCxWv11duBWUpAsnUNexPFsFmEjic7wJvx291iHokrKIq7Ue4TUIqNq0c8ZRmHyTFC+c9RjyO6ouVX64T7h1i4wtDFjlFyrPBipGJenhlJsk93KnuWoSSvmW1Ndz5Mg8Kx09oOrW/JXozjs8Q9sj8GlDQrutv7u8bYX8udL7LG9SHxlBr+ZopKqDA4YlAsZLvq6tEh8cj2w/1hvoU+E/Ei4kP6CIDC0G2iGCsSZGD5lwNHfMZlYav0bMCehB0GsWXFEWz01f7Vkv5rhRL03ODCRjxv3S9o4+OQPIma8W379zYfJ0KeWDUUCraPI7E23RaeyPSzsk2sCzusKlJo3ZQs3sban4uIqjNcN0M44H/of0s+mhF8r/Bnn8QQ2oM7nZASbHKSnuMXtX5+Fht+aC4O1df7E4ICnvRoUdxk0gGtIe1RCKfJc/mcYZZkOaApbyZ+nQLKAs1rCfw0P24QNCsSra4UrtHLX70xs8hR15wJWqCRaDkeCM0Ow2LHjer8/MWwT41wcTih0q4bPtvZlju9X6WgdboYlxA8NSmE=
```

Comment:
Seems that every repo has its own secret token, so I need someones help with that.

Reply:
Let's skip it for now – it will be different once we move it to percona org.

Reply:
Ok